### ISSUE TITLE
Templates system

### ISSUE THREAD

**Issue by voylinux** [APSL/puput#42]
This one looks more like a Wagtail improvement for me, but the current way to change the template is a little bit tricky.

If I understood well, you need to copy the whole puput folder and then change everything in the new folder.
Do you have any idea on a system to be able (for example) to create some Puput blog templates that could be installed by people who just want a blog?

**Comment by marctc** [comment]
Yeah, is an issue that I also have in mind. I think that we could add a "template_name" field on BlogPage model to select the name of the template that you previously created. If you have in mind a template system like Wordpress maybe this will be a Wagtail improvement (and interesting too).

**Comment by voylinux** [comment]
Ok, maybe not a number one priority but I will think about this one because this could be a game changer feature.

**Comment by shikolay** [comment]
+1

**Comment by StrattonStudios** [comment]
Yo!

I am working to refactor the templates structure down in an effort to make template extending easier. I am following the aldryn-newsblog layout where the base.html is semantic and free of <link> tags. If anyone is interested i can fork or open a pull request for review.

Thanks,
Greg











If there is interest i can fork a repo, or open a pull request here.

Thanks,
Greg

**Comment by marctc** [comment]
Hi @StrattonStudios. As far as I know, nobody is working on this. All that you say sounds amazing to me. Go ahead!

**Comment by chinnu0505** [comment]
6 years late, looking for a guide on how to extend Wagtail/Puput templates. Just want to add a logo to the blog navbar. Any links/tutorials I could check out?
